### DISCUSSION TITLE
Migrating from React: Best approach for my page

### DISCUSSION THREAD

**Discussion by sasi-wic** [vercel/next.js#22898] [Help]
We have a course description page with (at max 10 courses)

- 10 react components (rarely updated) - (SSG??)
- One component where we have dynamic data, pricing and batches (daily or weekly updated) (SSR??)

As ours is a ed-tech platform, SEO is very important for us.

- If we go with SSG we loose the ability to show updated pricing.
-  If we go with SSR our FFB will be higher and most of the data will not be changed frequently.

Any suggestions or inputs welcome.

**Comment by ludofischer** [comment]
You could go with SSG and fetch the updated data client-side from that single dynamic component (send an AJAX request).
